tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -83,7 +83,8 @@ spec:
           for DIGEST in $(jq -r ".layers[].digest" manifest.json)
           do
               FILE=${DIGEST#sha256:}
-              tar -xzvf "$FILE"
+              echo "Extracting $FILE..."
+              tar -xzvf "$FILE" --no-same-owner --no-same-permissions --strip-components=1
```

Comment:
Why is `--strip-components=1` needed? IIUC, it will remove the first dir from the path when extracting. So instead of e.g. `root/some/dir/and/file.txt` it will extract it as `some/dir/and/file.txt`. The path will never have the leading `/`, so why would we need to remove the root dir?

Reply:
yep! You are right. Removed. I thought of doing it to be extra cautious. But it is breaking the rest of the script

Reply:
Cool, thanks